python/HeaderService/hslib.py

```diff
@@ -265,6 +267,11 @@ def run_loop(self):
                     self.announce()
                     self.EndTelem.newEvent = False
                     LOGGER.info("------------------------------------------")
+                elif self.EndTelem.timeoutEvent:
+                    sys.stdout.flush()
+                    LOGGER.warning("Received timeout: {} Signal".format(self.name_end))
+                    LOGGER.info("Sending Error Event for timeout")
+                    self.timeoutError()
```

Comment:
Seems a bit dangerous to have the `!newEvent && !timeoutEvent` case fall through, but it appears that that can never happen.

Reply:
It should never happen, but I added and logger warning anyway.